### PR TITLE
A bunch of fixes for the install scripts...

### PR DESCRIPTION
**Pull request by KevinH-MS**

### CODE REVIEW

**Comment by KevinH-MS** [comment]
FYI @TyOverby

File: src/Test/Perf/util/ngen.csx [hunk 1]
```diff
@@ -4,24 +4,12 @@
...
-var ngen64 = Path.GetFullPath(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), @"Microsoft.NET\Framework64\ngen.exe");
-
-void NGen(string file, bool x86Only = false)
-{
-    // We'll have to remove the "fake sign" bit in order to ngen.
-    // Note: "FakeSign -u" may fail for reasons that should not prevent us from trying to ngen
-    // (for instance, if the binary was not "fake signed" to begin with).
-    ShellOut(fakeSign + " -u " + destinationFile);
-
-    var result = ShellOut(ngen, $"install {destinationFile} /nodependencies");
```
Comment (KevinH-MS) [inline src/Test/Perf/util/ngen.csx:24]: It looks like FakeSign is no longer needed (for reasons I don't entirely understand...but oh well...)

File: src/Test/Perf/infra/install.csx [hunk 2]
```diff
@@ -48,29 +50,29 @@ if (!result.Succeeded)
 {
     return result.Code;
 }
-result = ShellOut(devenv, "/resetsettingsfull Default.vssettings /command \"File.Exit\"");
+result = ShellOut(devenv, $"/resetsettingsfull {Path.Combine(sourceFolder, "Default.vssettings")} /command \"File.Exit\"");
```
Comment (TyOverby) [inline src/Test/Perf/infra/install.csx]: Consider using ShellOutVital

File: src/Test/Perf/util/ngen.csx [hunk 3]
```diff
@@ -4,24 +4,12 @@
...
+    var result = ShellOut(ngen, $"install {file} /nodependencies");
```
Comment (TyOverby) [inline src/Test/Perf/util/ngen.csx]: Consider using ShellOutVital

File: src/Test/Perf/util/ngen.csx [hunk 4]
```diff
@@ -29,10 +17,12 @@ void NGen(string file, bool x86Only = false)
 
     if (!x86Only)
     {
-        result = ShellOut(ngen64, $"install {destinationFile} /nodependencies");
+        result = ShellOut(ngen64, $"install {file} /nodependencies");
```
Comment (TyOverby) [inline src/Test/Perf/util/ngen.csx]: Consider using ShellOutVital

**Comment by TyOverby** [comment]
Aside from a few nits, this looks good!  Have you tried pulling this into automation.csx and running it on your machine?

**Comment by KevinH-MS** [comment]
Updated to use `ShellOutVital`.

I haven't run it through `automation.csx`.  That's next.

**Comment by TyOverby** [comment]
Cool, I'll pull this as it is!

**Pull request merged**